site/docs/status.md

```diff
@@ -0,0 +1,362 @@
+| binary         | Y    | Y         | Y    | Y  |
+
+## Data File Formats
+
+| Format  | Java | PyIceberg | Rust | Go |
+|---------|------|-----------|------|----|
+| Parquet | Y    | Y         | Y    | Y  |
+| ORC     | Y    | N         | N    | N  |
+| Puffin  | Y    | N         | N    | N  |
+
+## File IO
+
+| Storage              | Java | PyIceberg | Rust | Go |
+|----------------------|------|-----------|------|----|
+| Local Filesystem     | Y    | Y         | Y    | Y  |
+| Hadoop Filesystem    | Y    | Y         | Y    | Y  |
+| Aws S3               | Y    | Y         | Y    | Y  |
+| Google Cloud Storage | Y    | Y         | Y    | Y  |
+| Memory Fs            | Y    | Y         | Y    | Y  |
+
+## Table Update Operations
+
+### Table Spec V1
+
+| Operation                   | Java | PyIceberg | Rust | Go |
+|-----------------------------|------|-----------|------|----|
+| Update schema               | Y    | N         | Y    | N  |
+| Update partition spec       | Y    | N         | Y    | N  |
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | Y         | N    | N  |
+| Rewrite files               | Y    | Y         | N    | N  |
+| Rewrite manifests           | Y    | Y         | N    | N  |
+| Overwrite files             | Y    | Y         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
+| Delete files                | Y    | N         | N    | N  |
+| Update statistics           | Y    | N         | N    | N  |
+| Update partition statistics | Y    | N         | N    | N  |
+| Expire snapshots            | Y    | N         | N    | N  |
+| Manage snapshots            | Y    | N         | N    | N  |
+
+### Table Spec V2
+
+| Operation                   | Java | PyIceberg | Rust | Go |
+|-----------------------------|------|-----------|------|----|
+| Update schema               | Y    | Y         | N    | N  |
+| Update partition spec       | Y    | Y         | N    | N  |
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | Y         | N    | N  |
+| Rewrite files               | Y    | Y         | N    | N  |
+| Rewrite manifests           | Y    | Y         | N    | N  |
+| Overwrite files             | Y    | Y         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
+| Delete files                | Y    | Y         | N    | N  |
+| Update statistics           | Y    | N         | N    | N  |
+| Update partition statistics | Y    | N         | N    | N  |
+| Expire snapshots            | Y    | N         | N    | N  |
+| Manage snapshots            | Y    | N         | N    | N  |
+
```

Comment:
With PyIceberg, I've seen many users struggling to figure out why they are running into commit failures because reliability features have yet to be implemented in the subproject. 

Hence, I think it would help the community tremendously if we could also host information regarding the implementation of these features on the status page.

These are the [reliability features listed in this section](https://iceberg.apache.org/docs/1.5.2/reliability/#reliability) like isolation levels, and commit retries/concurrent writes support

Reply:
> As with other part such version history, it's more like a feature of time travel, maybe we should add features like incremental planning, incremental read, time travel into table read part?

Yes, I think that makes sense! I think that would be a good way to organize that information.

> As iceberg spec only supports serializable ioslation level, should we really need to mention this?

Now that you mention it, I think it would be helpful to update the documentation so that we explicitly describe the way Iceberg handles `snapshot` and `serializable` isolations are defined more clearly on the feature page (I can take a stab at that). I think this will help the language implementations correctly implement these behaviors consistently across.

The [Java API](https://iceberg.apache.org/javadoc/1.7.1/org/apache/iceberg/IsolationLevel.html) supports two isolation levels: serializable and snapshot isolation levels, and these modes dictate the type of commits that are allowed to be written on subsequent retries.

Do you think it would make sense to package these features as `commit retries` on the status page? Without this feature, the subprojects all fail on the first attempt if the metadata location of the table has changed since the beginning of the commit.

Reply:
Sounds good to me @liurenjie1024